resume.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>wommy.io - resume</title>
+
+		<link rel='stylesheet' href='style.css'>
+		<style>
+			#main {
+				margin: 1rem;
+			}
+			h2 {
+				border-bottom: 1px solid silver;
+				padding-bottom: .3rem;
+			}
+			.margin{
+				margin: 1rem 0;
+			}
+			.indent{
+				padding-left: 1.25rem;
+			}
+			.highlight{
+				background-color: gainsboro;
+				border-radius: 3px;
+				padding: .2em .4em;
+				font-size: 85%;
+				font-family: monospace;
+			}
+		</style>
+	</head>
+	<body id='resume'>
+		<nav>
+			<a href='./index.html'>home</a>
+		</nav>
+		<div id='main'>
+			<h1 style="display:none;">resume</h1>
+			<div id='content'>
```

Comment:
`#content` isn't used in your css, why is it here?

Reply:
more boilerplate, if i later choose to do something on it from the global stylesheet

Reply:
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it